.env.docker.example

```diff
@@ -1,9 +1,10 @@
 # DB settings
+# Ensure these match with docker-compose.yml
 POSTGRES_DB=metadeploy
 POSTGRES_USER=postgres
-POSTGRES_PASSWORD=
-# General settings
+POSTGRES_PASSWORD=sample_db_password
```

Comment:
Salesforce security team requires us to not include passwords in the repository, even if it's a sample password for a local environment.